drivers/lock.py

```diff
@@ -23,6 +23,9 @@
 
 VERBOSE = True
 
+# Still just called "running" for backwards compatibility
+LOCK_TYPE_GC_RUNNING = "running"
+LOCK_TYPE_ISCSIADM_RUNNING = "isciadm_running"
```

Comment:
This is fine but we will get a slightly odd lock name of `iscsiadm.iscsiadm_running` as a result.

Reply:
I'm fine with that. If in the future we encounter a lock named `newthing.iscsiadm_running` we'll know where to look for the cut & paste error...